plugin-packs/postcss-preset-env/src/index.js

```diff
@@ -10,20 +10,38 @@ import getOptionsForBrowsersByFeature from './lib/get-options-for-browsers-by-fe
 import { pluginIdHelp } from './lib/plugin-id-help';
 import { pluginHasSideEffects } from './lib/plugins-with-side-effects';
 
+const DEFAULT_STAGE = 2;
+const OUT_OF_RANGE_STAGE = 5;
+
+const logDebug = msg => console.log('[postcss-preset-env] -> %s', msg);
+
 const plugin = opts => {
 	// initialize options
-	const features = Object(Object(opts).features);
+	const options = Object(opts);
+	const features = Object(options.features);
+	const debug = options.debug ? logDebug : () => {};
 	const featureNamesInOptions = Object.keys(features);
-	const insertBefore = Object(Object(opts).insertBefore);
-	const insertAfter = Object(Object(opts).insertAfter);
-	const browsers = Object(opts).browsers;
-	const stage = 'stage' in Object(opts)
-		? opts.stage === false
-			? 5
-			: parseInt(opts.stage) || 0
-		: 2;
-	const autoprefixerOptions = Object(opts).autoprefixer;
-	const sharedOpts = initializeSharedOpts(Object(opts));
+	const insertBefore = Object(options.insertBefore);
+	const insertAfter = Object(options.insertAfter);
+	const browsers = options.browsers;
+	let stage = DEFAULT_STAGE;
+
+	if (typeof options.stage !== 'undefined') {
+		if (options.stage === false) {
+			stage = OUT_OF_RANGE_STAGE;
+		} else {
+			stage = parseInt(options.stage, 10) || 0;
```

Comment:
Should we add a lint rule to enforce adding the radix in `parseInt` ?

Reply:
Not sure. This was something automatic. Given the radix is 10 by default I can just remove it. Do you have any opinions on this?

Reply:
radix should always be added as it is not actually 10 by default :)
So your code is more correct 🎉 

I was wondering if we should add a lint rule to make sure it is always `parseInt(foo, 10)`;

Reply:
Added the lint rule, apparently only this file used to have `parseInt(foo)`.
So no other plugins needed fixes.

Reply:
Thanks!